airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/utils.py

```diff
@@ -11,6 +11,7 @@
 from typing import Dict
 
 import jsonschema
+import pandas as pd
```

Comment:
Is there something we can do to avoid using such heavy lib as `pandas` here? 
I can see that it was used to convert the `date` to `datetime` and to other formats.

Reply:
pandas responsible for transforming yearsWeeks to a date here, but not simple `date` to `datetime`.

Reply:
That's fine, I'm curious if this could be achieved using some `pendulum` or similar, something that is less heavy.

Reply:
you always have another solution